src/CarbonNetworks.jl

```diff
@@ -9,6 +9,7 @@ using LightGraphs
 using LinearAlgebra
 using SimpleWeightedGraphs
 using Zygote
+using Plots
```

Comment:
Let's not add `Plots` directly to the package. `Plots` takes a long time to compile. Instead, it is recommend we use `RecipesBase` (https://github.com/JuliaPlots/RecipesBase.jl).

Reply:
@degleris1 I am not sure I totally understand how this works. It looks like a macro/python decorator. Can you clarify what changes are required? e.g. in `plot_sensitivity_check` in `dyn_sensitivity.py`?